manifests/b/Bloodstrike/pc/1.1/Bloodstrike.pc.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Bloodstrike.pc

```

```suggestion
PackageIdentifier: NetEase.BloodStrike
```
- Please update this file's contents, name, and path with any changes to the `PackageIdentifier` above.